minikube.pro

```diff
@@ -12,6 +12,7 @@ HEADERS       = src/window.h \
                 src/operator.h \
                 src/paths.h \
                 src/progresswindow.h \
+                src/serviceView.h \
```

Comment:
The C++ filename convention is that it should be all lower case.

- [Currently all file names are all-lowercase, even though class names contain capital letters.](https://manual.gromacs.org/5.1.1/dev-manual/naming.html#:~:text=Currently%20all%20file%20names%20are%20all%2Dlowercase%2C%20even%20though%20class%20names%20contain%20capital%20letters.)
- [Filenames should be all lowercase](https://google.github.io/styleguide/cppguide.html#Naming:~:text=Filenames%20should%20be%20all%20lowercase)